src/sentry/static/sentry/app/views/events/utils/eventsRequest.tsx

```diff
@@ -15,21 +16,27 @@ import SentryTypes from 'app/sentryTypes';
 
 import LoadingPanel from '../loadingPanel';
 
-type RenderProps = {
-  loading: boolean;
-  reloading: boolean;
-  errored: boolean;
-
+type TimeSeriesData = {
   // timeseries data
   timeseriesData?: Series[];
   allTimeseriesData?: EventsStatsData;
   originalTimeseriesData?: EventsStatsData;
-  timeseriesTotals?: object;
+  timeseriesTotals?: {count: number};
   originalPreviousTimeseriesData?: EventsStatsData | null;
   previousTimeseriesData?: Series | null;
   timeAggregatedData?: Series | {};
 };
 
+type LoadingProps = {
```

Comment:
I think I'd give it another name.... `LoadStatus` for example

Reply:
Thanks. I'll rename this.

Reply:
Done in https://github.com/getsentry/sentry/pull/17334/commits/88cb0f82f1a9b9164c72bdec6bf7be21e1ef01ea